docs/sql-data-sources-csv.md

```diff
@@ -55,7 +55,7 @@ Data source options of CSV can be set via:
 <table class="table">
   <tr><th><b>Property Name</b></th><th><b>Default</b></th><th><b>Meaning</b></th><th><b>Scope</b></th></tr>
   <tr>
-    <td><code>sep</code></td>
+    <td><code>delimiter</code></td>
```

Comment:
Either sep or delimiter works fine. Let's probably fix the documentation that `delimiter` is an alias of `sep`.

Reply:
in spark 3.3.0, the `sep` configure  does not work fine in reading CSV format, but it works well for writing CSV.
And the `delimiter ` configure works well in read/write for CSV format.

Reply:
Could you show an example, please.